src/main/xerus/monstercat/tabs/TabSound.kt

```diff
@@ -22,7 +22,7 @@ class TabSound : VTab() {
 	
 	init {
 		add(CheckBox("Enable Equalizer").bind(Settings.ENABLEEQUALIZER))
-		Player.activePlayer.listen { Platform.runLater(::updateEQBox) }
+		Player.activePlayer.listen { onFx { updateEQBox() } }
```

Comment:
You can still use the method reference in `onFx` if I'm not mistaken ;)

Reply:
It works but produces a compiler warning. "The reference is unused"

Reply:
And no, it doesn't work, it doesn't invoke the function. You need to call .invoke() on it or run it normally with arguments bracket

Reply:
I think you are trying to create a lambda containing a method reference. But you mustn't create a lambda at all: `onFx(::updateEQBox)`

Reply:
Exactly, now I understand.

Reply:
I find the regular way to be more verbose and easier to understand. Your take on this ? Should it stay like this or should I use the double-colon ?

Reply:
I prefer the shorter way. That the longer way is easier for you to understand is probably because you are not used to working with method references.

Reply:
89ac926e188193ba567a503caabbcf739bb1f813